distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Of_Day.enso

```diff
@@ -217,8 +217,7 @@ type Time_Of_Day
 
              example_plus = Time_Of_Day.new + 3.seconds
     + : Duration -> Time_Of_Day
-    + self amount = if amount.is_date then Error.throw (Time_Error_Data "Time_Of_Day does not support date intervals") else
-        Time_Utils.time_adjust self Time_Utils.AdjustOp.PLUS amount.internal_duration
+    + self amount = self.plus_builtin amount
```

Comment:
Any particular reason why we should prefer builtins rather than calling Java code? This is a general question, also to @JaroslavTulach

Reply:
Builtins are necessary when we need to deal with `TruffleObject` and use a bit of partial evaluation. Otherwise it is better if the libraries stick with classical (hosted) Java interop.